frontend/app/src/people/widgetViews/organization/EditOrgModal.tsx

```diff
@@ -152,46 +84,89 @@ const EditOrgModal = (props: EditOrgModalProps) => {
   };
 
   const [selectedImage, setSelectedImage] = useState<string>(org?.img || '');
+  const [rawSelectedFile, setRawSelectedFile] = useState<File | null>(null);
   const fileInputRef = useRef<HTMLInputElement | null>(null);
 
+  const onSubmitEditOrg = async (body: any) => {
+    if (!org) {
+      addToast('Invalid organization update', 'danger');
+      return;
+    }
+    setLoading(true);
+    try {
+      let img = '';
+      const formData = new FormData();
+      if (rawSelectedFile) {
+        formData.append('file', rawSelectedFile);
+        const file = await main.uploadFile(formData);
+        if (file && file.ok) {
+          img = await file.json();
+        }
+      }
+
+      const newOrg = {
+        id: org.id,
+        uuid: org.uuid,
+        name: body.name || org.name,
+        owner_pubkey: org.owner_pubkey,
+        img: img || org.img,
+        created: org.created,
+        updated: org.updated,
+        show: body?.show !== undefined ? body.show : org.show,
+        bounty_count: org.bounty_count,
+        budget: org.budget
+      };
```

Comment:
I trying to think how this will work because we are uploading the file to the meme server through the backend. 

Are you saying we have a function that uploads to meme the server?? and returns a string we could use??